src/main/java/fr/techad/sonar/gerrit/ReviewUtils.java

```diff
@@ -119,36 +129,28 @@ public static int maxLevel(ReviewInput reviewInput) {
         return lvl;
     }
 
+    private static String prefixKey(String prefix, String key) {
+        return prefix + "." + key;
+    }
+
+    public static String issueMessage(String originalMessage, Settings settings, Issue issue) {
+        HashMap<String, Object> valueMap = new HashMap<>();
+        valueMap.put(prefixKey(ISSUE_PREFIX, ISSUE_IS_NEW), issue.isNew());
+        valueMap.put(prefixKey(ISSUE_PREFIX, ISSUE_RULE_KEY), issue.ruleKey());
+        valueMap.put(prefixKey(ISSUE_PREFIX, ISSUE_SEVERITY), issue.severity());
+        valueMap.put(prefixKey(ISSUE_PREFIX, ISSUE_MESSAGE), issue.message());
+        return substituteProperties(originalMessage, settings, valueMap);
+    }
+
 	public static String substituteProperties(String originalMessage, Settings settings) {
-	    String substitutedString = originalMessage;
```

Comment:
That’s fine : I didn’t know StringUtils provided this functionality (would have save me some time).